logicrunner/rpc_methods.go

```diff
@@ -189,18 +189,21 @@ func (m *executionProxyImplementation) RouteCall(
 	if req.Saga {
 		// Saga methods are not executed right away. LME will send a method
 		// to the VE when current object finishes the execution and validation.
+		if outReqInfo.Result != nil {
```

Comment:
why is it an error? maybe something was broken in between or discarded by heavy

Reply:
If we are processing a request that creates a saga and saga is already closed something is VERY wrong with the system. There is no valid scenario when it can happen. Personally I suggested to `panic()` in this case.